server/src/compression.rs

```diff
@@ -398,14 +398,20 @@ mod tests {
     use super::*;
 
     use datafusion::arrow::array::UInt8Array;
+    use rand::{thread_rng, Rng};
 
+    use crate::compression::test_util as comp_test_util;
```

Comment:
For consistency import `DataType` directl.

Reply:
Fixed.